src/cr/cube/matrix/assembler.py

```diff
@@ -1096,4 +1096,20 @@ class _SortRowsByMarginalHelper(_RowOrderHelper):
     a matrix dimension has an opposing dimension.
     """
 
-    pass
+    @lazyproperty
+    def _element_values(self):
+        raise ValueError("_element_values not implemented yet ")
```

Comment:
I usually use `NotImplementedError` (by itself) for fields I haven't gotten to yet. The naming seems apt :) I generally don't bother with a message since I can tell what's not implemented by the error message location and I'm going to implement it in a second anyway.

I do however make it a practice to write the docstring as soon as I add the method. When I (used to) wait I often ended up not remembering it. It's also a key opportunity to make sure I have a clear idea what this method is going to do for me (its contract). That's good timing because if I change my mind about what (or how much) it should do (which happens more than one might expect) then I can change the caller without having to revert code I've already written a test for a committed.